develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -0,0 +1,122 @@
+
+The purpose of this article is to help you:
+
+- Learn how to update runtime dependencies to integrate a new pallet
+- Understand how to configure pallet-specific Rust traits to enable the pallet's functionality
+- Grasp the entire workflow of integrating a new pallet into your runtime
+
+## Configuring Runtime Dependencies
+
+For Rust programs, this configuration is defined in the `Cargo.toml` file, which specifies the settings and dependencies that control what gets compiled into the final binary. Since the Polkadot SDK runtime compiles to both a native binary (which includes standard Rust library functions) and a Wasm binary (which does not include the standard Rust library), the `runtime/Cargo.toml` file manages two key aspects:
+
+- The pallets are imported as dependencies for the runtime, including their locations and versions
+- The features in each pallet that should be enabled when compiling the native Rust binary. By enabling the standard (`std`) feature set from each pallet, you ensure that the runtime includes the functions, types, and primitives necessary for the native build, which are otherwise excluded when compiling the Wasm binary
+
+!!! note
+    For information about adding dependencies in `Cargo.toml` files, see [Dependencies](https://doc.rust-lang.org/cargo/guide/dependencies.html){target=\_blank} page in the Cargo documentation. For information about enabling and managing features from dependent packages, see [Features](https://doc.rust-lang.org/cargo/reference/features.html){target=\_blank} section in the Cargo documentation.
+
+## Dependencies for a New Pallet
+
+To add the dependencies for a new pallet to the runtime, you need to modify the `Cargo.toml` file by adding a new line into the `[workspace.dependencies]` section with the pallet you want to add. Ensure that you stick to the following format:
+
+```toml
+pallet-example = { version = "4.0.0-dev", default-features = false }
+```
+
+This line imports the pallet-example crate as a dependency and specifies the following:
+
+- `version` - the specific version of the crate to import
+- `default-features` - determines the behavior for including pallet features when compiling the runtime with standard Rust libraries
+
+Next, add this dependency to the `[dependencies]` section of the `runtime/Cargo.toml` file, so it inherits from the main `Cargo.toml` file:
+
+```toml
+pallet-indices.workspace = true
+```
+
+To enable the `std` feature of the pallet, add the pallet to the following section:
+
+```toml
+[features]
+default = ["std"]
+std = [
+    ...
+    "pallet-example/std",
+    ...
+]
+```
+
+This section specifies the default feature set for the runtime, which includes the `std` features for each pallet. When the runtime is compiled with the `std` feature set, the standard library features for all listed pallets are enabled for more details on how the runtime is compiled as both a native binary (using `std`) and a Wasm binary (using `no_std`).
```

Comment:
> ....for more details on how the runtime is compiled as both a native binary (using `std`) and a Wasm binary (using `no_std`)... 

was the intention to link to another page?

```suggestion
This section specifies the default feature set for the runtime, which includes the `std` features for each pallet. When the runtime is compiled with the `std` feature set, the standard library features for all listed pallets are enabled for more details on how the runtime is compiled as both a native binary (using `std`) and a Wasm binary (using `no_std`).
```

Reply:
yes, you are right. I missed that one